config.json

```diff
@@ -224,6 +224,41 @@
         "default"   : "sum"
     },
     {
+        "tab"           : "Header",
+        "key"           : "header.megamenu_levels",
+        "label"         : "Category levels in mega menu",
+        "type"          : "dropdown",
+        "possibleValues":
+        {
+            "1"         : "inactive",
+            "2"         : "2",
+            "3"         : "3",
+            "4"         : "4"
+        },
+        "default"       : "1"
+    },
+    {
+        "tab"           : "Header",
+        "key"           : "header.megamenu_items_stage_1",
+        "label"         : "Number of categories in level 2",
```

Comment:
Wie wird bestimmt, welche 30 Kategorien in der zweiten Ebene angezeigt werden? Über die Position der Kategorie?

Reply:
ja

alles geht immer darüber